microsoft-edge/webview2/release-notes/index.md

```diff
@@ -97,33 +95,200 @@ description
 ###### Runtime and SDK
 
 * item
-
 * item
 
 
 <!-- ---------- --
 ###### Runtime-only
 
 * item
-
 * item
 
 
 <!-- ---------- --
 ###### SDK-only
 
 * item
-
 * item
 
 -->
 
 
-<!-- end of Oct 2024 Release SDK -->
-<!-- end of Oct 2024 Prerelease SDK -->
+<!-- end of Dec 2024 Release SDK -->
+<!-- end of Dec 2024 Prerelease SDK -->
 <!-- end of template -->
 
 
+<!-- ====================================================================== -->
+## 1.0.2903.40
+
+Release Date: November 18, 2024
+
+[NuGet package for WebView2 SDK 1.0.2903.40](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2903.40)
+
+For full API compatibility, this Release version of the WebView2 SDK requires WebView2 Runtime version 131.0.2903.40 or higher.
+
+
+<!-- ------------------------------ -->
+#### Promotions
+
+The following APIs have been promoted to Stable and are now included in this Release SDK.
+
+
+<!-- ---------- -->
+###### Control whether the screen capture UI is shown (ScreenCaptureStarting event)
+
+Added a new ScreenCaptureStarting event.  This event is raised whenever the WebView2 and/or iframe that corresponds to the CoreWebView2Frame (or to any of its descendant iframes) requests permission to use the Screen Capture API before the UI is shown.  The app can then block the UI from being displayed, or allow the UI to be displayed.
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+<!-- todo: global: un-link parent if had members already -->
+
+* `CoreWebView2` Class:
+   * [CoreWebView2.Cancel Property](/dotnet/api/microsoft.web.webview2.core.corewebview2.cancel?view=webview2-dotnet-1.0.2903.40&preserve-view=true)<!-- todo: copied from rt, ok? -->
```

Comment:
I added these 4 properties b/c they are in WinRT tab.  ok?
Build report claims 404 only for these 4 Property members that i manually constructed.
ScreenCaptureStarting Event seems ok.

@sivMSFT @victorhuangwq